app/views/layouts/application.html.erb

```diff
@@ -6,30 +6,57 @@
 
     <title><%= content_for?(:title) ? yield(:title) : "Caesar" %></title>
 
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
```

Comment:
Hi Amy, I'd forgotten to push the changes from yesterday. We ended up importing a customisable version of Bootstrap and jQuery with Rails so downgraded to Bootstrap 3. I think that addresses your comment?